### DISCUSSION TITLE
Using Cuda, gpu for (ultralytics file example)

### DISCUSSION THREAD

**Discussion by Touville01** [roboflow/supervision#1234] [Q&A] [answered]
I I'm not able to run the example with the argument device --cuda. I've a Nvidia Gforce card ... do you have any advices. Should I set up something before ? Thank you

**Comment by SkalskiP** [comment, marked as answer]
Hi, @Touville01 👋🏻 Can you confirm that you have installed PyTorch with GPU acceleration?
